modules/apps/object/object-test/src/testFunctional/tests/ObjectFieldsAutoIncrement.testcase

```diff
@@ -0,0 +1,23 @@
+@component-name = "portal-object"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Object";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		ObjectAdmin.deleteAllCustomObjectsViaAPI();
+
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		if (${testLiferayVirtualInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+}
```

Comment:
You don't need to make a separate commit to create the testcase file, you can just do that while adding the first test

Reply:
You don't need to make this change for this PR, just something to keep in mind moving forward.

Reply:
Ok, info saved 🧠.